src/pyaddlicense/main.py

```diff
@@ -424,6 +433,8 @@ def create_license_header(path: Path, templated_license: str) -> Optional[str]:
 
 _INDICATOR_STRINGS = [
     "copyright",
+    "license",
```

Comment:
I see one minor issue with the implementation; e.g. if you have a top level docs block for a file that has anything to do with licenses, etc. 😄 But that seems like a rare edge case.